src/datachain/cli/__init__.py

```diff
@@ -108,15 +115,23 @@ def handle_command(args, catalog, client_config) -> int:
     return 1
 
 
-def handle_cp_command(args, catalog):
-    catalog.cp(
-        args.sources,
-        args.output,
-        force=bool(args.force),
-        update=bool(args.update),
-        recursive=bool(args.recursive),
-        no_glob=args.no_glob,
-    )
+def handle_cp_command(args, catalog: "Catalog"):
+    from datachain.config import Config
+
+    config = Config().read().get("studio", {})
+    token = config.get("token")
+    local = True if not token else args.local
```

Comment:
This is what I am not quite satisfied about. How should we distinguish between the call that should use the cp from catalog or through studio? cc. @shcheklein 

https://github.com/iterative/datachain/pull/1221#issuecomment-3057437853

Reply:
can we always use catalog and just notify Studio if token is set about changes after it is done?

Reply:
yes, I'm just trying to simplify the scope (let me know if that doesn't make much difference). The request was to be able to have audit log in Studio (less about using Studio managed credentials).

We can add additional explicit feature `--studio-cloud-auth` to enable actual cloud credentials.

I think even if we keep we should decouple audit log from the actual mechanism how we actually perform the operation.

Reply:
Yes, but I don't know, I don't feel comfortable with having a call to add activity log when the activity is performed completely locally. 

We already had `datachain cp` for activities user want to make the changes using local credentials. I think keeping that as it is and using following umbrella or following structure justify the change much more:
```
datachain studio cp
datachain studio rm
datachain studio mv
```

to explicitly mention that the activities are being performed through Studio.